ISHPullUp/ISHPullUpViewController.h

```diff
@@ -190,6 +190,20 @@ typedef struct ISHPullUpAnimationConfiguration ISHPullUpAnimationConfiguration;
 /// The layout mode to be used for the bottom view controller. Default is ISHPullUpBottomLayoutModeShift.
 @property (nonatomic) ISHPullUpBottomLayoutMode bottomLayoutMode;
 
+/// YES if bottom is hidden outside of view
+@property (nonatomic) BOOL bottomHidden;
```

Comment:
Maybe make this `readonly` to make `setBottomHidden:animated:` more discoverable?